debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -6,6 +6,23 @@
 
 package io.debezium.connector.postgresql;
 
+import io.debezium.annotation.ThreadSafe;
+import io.debezium.connector.postgresql.connection.ReplicationConnection;
+import io.debezium.connector.postgresql.connection.ReplicationStream;
+import io.debezium.connector.postgresql.proto.PgProto;
+import io.debezium.data.Envelope;
+import io.debezium.relational.Column;
+import io.debezium.relational.Table;
+import io.debezium.relational.TableId;
+import io.debezium.relational.TableSchema;
+import io.debezium.util.LoggingContext;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.postgresql.geometric.PGpoint;
+
```

Comment:
Yes, my apologies - my IDE was 'optimizing' imports and I had to manually undo that damage. Will not happen again since I've disabled that feature now.  Probably should have reverted to the version in git but I thought it would be quicker and harmless to put the imports back.